add llms mbti code, streamlit app, readme

Derekwang2002 opened the pull request:
date: 7.5

Derekwang2002 committed:
add llms mbti code, streamlit app, readme

7.5

Derekwang2002 committed:
refine README and requirement

Derekwang2002 committed:
Update README.md

yihong-eric-chen commented on README.md:
Add instructions and scripts to set up the environment for this project.

yihong-eric-chen commented on gpt-api-starter.ipynb:
Let users know how to create '.env.\n' and where to store the key in README

yihong-eric-chen commented on README.md:
Order to execute notebooks and python scripts matters! 'gpt-api-starter.ipynb' + 'mbti_questions.json' -> 'llms_mbti.json', then 'llms_mbti.json' can be further used in 'web.py'. Let users know the structure of the project and how to execute each script in order.

Derekwang2002 committed:
add gpt-api-starter.py, update requirements.txt

yihong-eric-chen commented on web.py:
Remove '\n', there can be 2 separate columns to display 'Question' and 'Answers'.
<img width="678" alt="image" src="https://github.com/datamonet/llm-personality/assets/115609230/a6537d34-7d69-43d7-a9b6-c368a180d685">

118020020 commented:
1. You may try to package the images in one folder for a clearer structure. You can refer to [this](https://backrightup.com/blog/how-to-organize-github-repositories/) for more details. But you do not need to be so complicated since it's a rather easy project. But ideally, it is better to let viewers know the function of each file once they see the repo.
2. For more about creating a nice readme file, you can refer to this [link](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/).

118020020 commented:
> date: 7.5

1. You may try to package the images in one folder for a clearer structure and consider using some tricks to better organize the repo. You can refer to [this](https://backrightup.com/blog/how-to-organize-github-repositories/) for more details. But you do not need to be so complicated since it's a rather easy project. Ideally, it is better to let viewers know the function of each file once they see the repo.
2. For more about creating a nice readme file, you can refer to this [link](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/).

CH-chuan commented:
1. You may try to package the images in one folder for a clearer structure and consider using some tricks to better organize the repo. You can refer to [this](https://backrightup.com/blog/how-to-organize-github-repositories/) for more details. But you do not need to be so complicated since it's a rather easy project. Ideally, it is better to let viewers know the function of each file once they see the repo.
2. For more about creating a nice readme file, you can refer to this [link](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/).

Derekwang2002 committed:
update folder structure

Derekwang2002 committed:
Update README.md

Add tutorials (install packages, use .env to store keys, functions of main files, how to locally run web.py)

Derekwang2002 committed:
Update file structure, readme, web.py

split the mbti question form in web.py to 3 columns. add res folder includes images, src folder includes code and json files.

Derekwang2002 committed:
Update README.md

Derekwang2002 committed:
Update README.md

Derekwang2002 committed:
Update README.md
